kvrocks.conf

```diff
@@ -587,15 +587,19 @@ rocksdb.target_file_size_base 128
 # allowed.
 rocksdb.max_write_buffer_number 4
 
-# Maximum number of concurrent background compaction jobs, submitted to
+# Maximum number of concurrent background jobs (compactions and flushes).
+# RocksDB automatically decides this based on the  value of max_background_jobs. For backwards compatibility we will set `max_background_jobs = max_background_compactions + max_background_flushes` in the case where user sets at least one of `max_background_compactions` or `max_background_flushes` (we replace -1 by 1 in case one option is unset).
+ rocksdb.max_background_jobs 4
+
+# DEPRECATED : Maximum number of concurrent background compaction jobs, submitted to
```

Comment:
You need to specify a reason for DEPRECATED.

e.g. `# DEPRECATED: the reason, and what users can do`